compiler/rustc_codegen_ssa/src/traits/type_.rs

```diff
@@ -86,16 +86,7 @@ pub trait DerivedTypeCodegenMethods<'tcx>:
     }
 
     fn type_has_metadata(&self, ty: Ty<'tcx>) -> bool {
```

Comment:
Maybe inline this method everywhere?

Reply:
Indeed. I don't think it is useful to keep as wrapper.

Reply:
Ah, I see

Reply:
Fixed now